iothub/device/src/Transport/Http/CustomHeaderConstants.cs

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-namespace Microsoft.Azure.Devices.Client
+namespace Microsoft.Azure.Devices.Client.Transport
```

Comment:
I'd like to not have namespaces match the folders. Unless the files in there already do, can we leave this the same as the root namespace?

Reply:
Sure I can revert this change